src/phoenix/server/api/routers/v1/spans.py

```diff
@@ -87,6 +93,7 @@ async def query_spans_handler(request: Request) -> Response:
                     project_name=project_name,
                     start_time=from_iso_format(payload.get("start_time")),
                     stop_time=from_iso_format(payload.get("stop_time")),
+                    limit=payload.get("limit"),
```

Comment:
don't you have to put a default here?

Reply:
i think this is to let ppl to send in None if they don't want the limit

Reply:
addressed in [83760a0](https://github.com/Arize-ai/phoenix/pull/3026/commits/83760a00bc7554c3ee73ccff931fdb5bbb24b990)